tuf/tuf.go

```diff
@@ -458,8 +491,7 @@ func (tr *Repo) InitRoot(root, timestamp, snapshot, targets data.BaseRole, consi
 	if err != nil {
 		return err
 	}
-	tr.Root = r
-	return nil
+	return tr.SetRoot(r)
```

Comment:
I originally thought I needed to call `SetRoot` because it sets the originalRootRole - I'm wondering now though whether `originalRootRole` should be a pointer and it should't `originalRootRole` if it's not set?  That way we don't accidentally save version 0.